content/docs/v1/overview/index.html

```diff
@@ -127,18 +127,18 @@ <h2 class="title"><a href="#cordova">Cordova/PhoneGap/Trigger.io</a></h2>
 
   <h2 class="title"><a href="#browser-support">Browser Support</a></h2>
 
-  <h3>Ionic 1</h3>
+  <h3>Ionic V1</h3>
```

Comment:
Maybe in this file it should be 1.x everywhere and 2.x - 3.x where Ionic 2 is? Since the following would be the grouping of the different versions:

- Ionic 1
- Ionic 2 - Ionic 3
- Ionic 4